REUSE.toml

```diff
@@ -6,6 +6,7 @@ version = 1
 [[annotations]]
 path = [
     "**/VERSION",
+    "**.md",
```

Comment:
Any reason for this update to the reuse file?

Reply:
https://github.com/open-edge-platform/edge-node-agents/actions/runs/14701689617/job/41252319501

Reply:
The previous run had failed one it.

Reply:
Merge #42 from mainline now.